src/util/message.h

```diff
@@ -36,7 +36,23 @@ enum class MessageVerificationResult {
     ERR_NOT_SIGNED,
 
     //! The message verification was successful.
-    OK
+    OK,
+
+    //
+    // BIP-322 extensions
```

Comment:
I'm unsure, but perhaps we should make a fresh `enum BIP322VerificationResult`?

Reply:
Oftentimes you won't know which type it is until you finish verifying, so unifying the two types seems sensible to me.